docs/decisions/0003-graphql-schema.rst

```diff
@@ -0,0 +1,28 @@
+3. Learner-portal graphql schema
+--------------------------------
+
+Status
+------
+Draft 
+
+Context
+-------
+When data coming in from the designer api changes, we don’t want to leave it up to Gatsby to infer the fields and create nodes based on incomplete information. Instead, we would like to explicitly name the types for graphql, as to determine the exact structures that queries can have.
```

Comment:
For newbs like me, can you add a line about why we don't want to leave it up to Gatsby?

Reply:
I added more of an explanation here -- let me know if I'm explaining myself properly about it or if it is still confusing.